### PR TITLE
Feature/audio only to social

### PR DESCRIPTION
**Pull request by mekya**
# Pull Request

### CODE REVIEW

File: src/main/java/io/antmedia/rest/StreamsSourceRestService.java [hunk 1]
```diff
@@ -159,6 +169,30 @@ public Result addSource(Broadcast stream) {
 	}
 
 
+
+	private void addSocialEndpoints(Broadcast streamSource, String socialEndpointIds) {
```
Comment (mekya) [inline src/main/java/io/antmedia/rest/StreamsSourceRestService.java]: Almost same function exists in BroadcastRestService class. This is code duplication. Is it possible to use the same  function in BroadcastRestService? @burak-58

File: src/main/java/io/antmedia/rest/StreamsSourceRestService.java [hunk 2]
```diff
@@ -60,7 +65,7 @@
 	@Consumes(MediaType.APPLICATION_JSON)
 	@Path("/addStreamSource")
 	@Produces(MediaType.APPLICATION_JSON)
-	public Result addStreamSource(Broadcast stream) {
+	public Result addStreamSource(Broadcast stream, @QueryParam("socialNetworks") String socialEndpointIds) {
```
Comment (mekya) [inline src/main/java/io/antmedia/rest/StreamsSourceRestService.java]: I think we need functional test code for this feature. What do you think? @burak-58

File: src/main/java/io/antmedia/rest/StreamsSourceRestService.java [hunk 3]
```diff
@@ -239,6 +273,15 @@ public Result updateCamInfo(Broadcast broadcast) {
 			}
 
 			result = getStore().editStreamSourceInfo(broadcast);
+
```
Comment (mekya) [inline src/main/java/io/antmedia/rest/StreamsSourceRestService.java]: Functional or unit test code may be needed for this block as well?

File: src/main/java/io/antmedia/rest/StreamsSourceRestService.java (hunk 1 above)
Comment (burak-58) [reply src/main/java/io/antmedia/rest/StreamsSourceRestService.java]: Absolutely true. It must be refactored to fix code duplication.

File: src/main/java/io/antmedia/rest/StreamsSourceRestService.java (hunk 2 above)
Comment (burak-58) [reply src/main/java/io/antmedia/rest/StreamsSourceRestService.java]: Yes, I think so.

File: src/main/java/io/antmedia/rest/StreamsSourceRestService.java (hunk 3 above)
Comment (burak-58) [reply src/main/java/io/antmedia/rest/StreamsSourceRestService.java]: For which block?

**Pull request merged**